docs/section19/19-25.md

```diff
@@ -0,0 +1,32 @@
+# Section 19 - Rule 19-25           
+**Schema Version:** 0.0.23    
+**Mandatory Rule:** True    
+**Rule ID:** 19-25         
+**Rule Description:** Baseline systems modeled with exhaust air energy recovery shall allow bypass for economizer control where applicable.      
```

Comment:
Please update rule description to say "bypass OR CONTROL HEAT RECOVERY SYSTEM to permit air economizer operation". Because this enumeration in the RMD does not necessarily mean bypass.

Reply:
Done